gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/DashboardsResource.java

```diff
@@ -62,7 +63,7 @@ public class DashboardsResource extends AbstractResource {
         )
     )
     @ApiResponse(responseCode = "500", description = "Internal server error")
-    public List<DashboardEntity> getDashboards(final @QueryParam("reference_type") DashboardReferenceType referenceType) {
+    public List<DashboardEntity> getDashboards(final @QueryParam("type") DashboardType type) {
```

Comment:
Do you think we could continue to accept "reference_type" as a query param, but linked to `DashboardType`?
And if both query param are set, `type` takes precedence over `reference_type`

WDYT?

Reply:
we can discuss it, but if I remember correctly, we decided to go for a breaking change to ease our life